c++/src/vector.cc

```diff
@@ -44,18 +45,26 @@ class Vector {
     return this->dataStore_[length_--];
   }
 
-  unsigned int length() const{
-    return this->length_;
-  }
+  void reserve(unsigned int reserveIndices) {
```

Comment:
I think `reserveIndices` should be `requestedCapacity` or something. The current name seems kind of weird, it would lead me to believe that I tell it which indices to reserve or something, like if I was supposed to give it a vector saying "yeah I'd like index 0, 5, and 9 please."